gobblin-azkaban/build.gradle

```diff
@@ -21,12 +21,20 @@ dependencies {
   compile project(":gobblin-utility")
   if (project.hasProperty('useHadoop2')) {
     compile project(":gobblin-yarn")
+    compile externalDependency.hadoopClientCommon
+    compile externalDependency.hadoopYarnCommon
+  } else {
+    compile group: 'org.apache.hadoop', name: 'hadoop-yarn-api', version: '2.3.0'
```

Comment:
Otherwise it won't build with hadoop1.
I'm going to put this class in the gobblin-yarn submodule so that we don't need to do this.